lib/bot.js

```diff
@@ -50,6 +55,16 @@ Bot.prototype.attachListeners = function() {
   this.client.addListener('error', function(error) {
     logger.error('Received error event from IRC', error);
   });
+
+  this.client.addListener('invite', function(channel, from, message) {
+    logger.debug('Received invite', channel, from, message);
+    if (this.opt.channels.indexOf(channel) > -1) {
```

Comment:
Is that possible to use in this case - channel here is just he name of the IRC channel, while channelMapping seems to take a mapping from the config file as the argument? I left it unchanged for now.